### PR TITLE
mark the copr make_srpm git dir as safe (git 2.35.2 security fix)

### PR DESCRIPTION
**Pull request by arso**
See [CVE-2022-24765](https://github.blog/2022-04-12-git-security-vulnerability-announced/)

Signed-off-by: Artur Socha <asocha@redhat.com>

### CODE REVIEW

File: .copr/Makefile [hunk 1]
```diff
@@ -19,6 +19,12 @@ installdeps:
 
 
 srpm: installdeps
+# explicity mark the copr generated git repo directory (which is done prior to the mock
+# call to the make_srpm and will be the current pwd) as safe for git commands
+git-safe:
```
Comment (sandrobonazzola) [inline .copr/Makefile:24]: Please add git-safe to .PHONY

**Review by sandrobonazzola — CHANGES_REQUESTED** [review]
Inline comment

File: .copr/Makefile [hunk 2]
```diff
@@ -19,6 +19,12 @@ installdeps:
 
 
 srpm: installdeps
```
Comment (sandrobonazzola) [inline .copr/Makefile:21]: I think you missed to delete above line `srpm: installdeps`

Comment (arso) [reply .copr/Makefile:21]: Right! Thanks Sandro.

Comment (arso) [reply .copr/Makefile]: fixed

**Review by sandrobonazzola — APPROVED** [review]
LGTM

**Pull request merged**